ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -482,6 +468,7 @@ public class RNTrackPlayer: RCTEventEmitter, AudioSessionControllerDelegate {
         }
 
         player.stop()
+        player.reset()
```

Comment:
This is a private function of SwiftAudio. Did you mean to put something else here? @dcvz